src/BinSkim.Rules/PERules/BA2006.BuildWithSecureTools.cs

```diff
@@ -54,6 +56,15 @@
 
         private const string MIN_XBOX_COMPILER_VER = "MinimumXboxCompilerVersion";
 
+        public static PerLanguageOption<Version> MinimumCCompilerVersion { get; } =
```

Comment:
>static [](http://example.com/codeflow?start=15&length=6)

Please follow the style convention in this file for options, which requires a line between each option.